physlr/physlr.py

```diff
@@ -537,9 +537,13 @@ def split_junctions_of_tree(prune_junctions, gin):
             if Physlr.args.verbose >= 3:
                 print("Junction:", junction, "Edges:", *edges, file=sys.stderr)
             # Keep the two incident edges with the largest weight, and remove the rest.
-            for u, v, _ in edges[2:-1]:
-                g.remove_edge(u, v)
-        return g
+            if keep_largest:
+                for u, v, _ in edges[2:-1]:
```

Comment:
shouldn't this code be
```suggestion
                for u, v, _ in edges[1:]:
```

Reply:
It keeps removes all the edges except the first two edges. so skipping 0 and 1 it should iterate from 2 to last. so [2:]. I'll fix it.

Reply:
It said `keep_largest` so I thought you only wanted the largest edge hence remove everything from 1 to last. So do you want to keep the two largest?

Reply:
the largest link between two of the branches, that includes two edges. yep.

Reply:
Yes, that's a bug. It should be `[2:]`. What's the effect of that bug?

Reply:
Also 😠 😭

Reply:
hmm looks like I misunderstood the intention behind the code. That's all

Reply:
@sjackman no effects. This part of the code was not in the pipeline.